testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestComments.java

```diff
@@ -50,51 +50,64 @@ public void testCommentTable()
                         "   format = 'RCBINARY'\n" +
                         ")",
                 COMMENT_TABLE_NAME);
-
         query(createTableSql);
+
+        String createTableSqlPattern = format("\\Q" +
+                        "CREATE TABLE hive.default.%s (\n" +
+                        "   c1 bigint\n" +
+                        ")\n" +
+                        "COMMENT 'old comment'\n" +
```

Comment:
Maybe we the pattern could be even simpler: `(?s).*COMMENT 'old comment'\n.*`

Reply:
I simplified. But not to that extreme. I want to be sure that comment is on table not on column